src/main/java/com/mercadopago/MercadoPagoConfig.java

```diff
@@ -0,0 +1,176 @@
+package com.mercadopago;
+
+import com.mercadopago.net.IHttpClient;
+import com.mercadopago.net.MPDefaultIHttpClient;
+import java.util.logging.Level;
+import org.apache.http.HttpHost;
+import org.apache.http.client.HttpRequestRetryHandler;
+
+public class MercadoPagoConfig {
+    private static final String DEFAULT_BASE_URL = "https://api.mercadopago.com";
+    private static final String CURRENT_VERSION = "1.11.0";
+    private static final String PRODUCT_ID = "BC32A7VTRPP001U8NHJ0";
+    private static final String CLIENT_NAME = "MercadoPago-SDK-Java";
+    private static final String TRACKING_ID = String.format("platform:%s,type:SDK%s,so;", getJavaVersion(System.getProperty("java.runtime.version")), CURRENT_VERSION);
```

Comment:
Ao invés de ter métodos estáticos publicos, deixe  a propriedade publica.

```suggestion
    public static final String CURRENT_VERSION = "1.11.0";
    public static final String PRODUCT_ID = "BC32A7VTRPP001U8NHJ0";
    public static final String CLIENT_NAME = "MercadoPago-SDK-Java";
    public static final String TRACKING_ID = String.format("platform:%s,type:SDK%s,so;", getJavaVersion(System.getProperty("java.runtime.version")), CURRENT_VERSION);
```

Reply:
Feito.